test/es-module/test-esm-loader-chaining.mjs

```diff
@@ -0,0 +1,300 @@
+import '../common/index.mjs';
+import fixtures from '../common/fixtures.js';
+import assert from 'node:assert';
+import { spawnSync } from 'node:child_process';
+
+const commonArgs = [
+  '--no-warnings',
+  '--input-type=module',
+  '-e',
+  'import fs from "node:fs"; console.log(fs)',
+];
+
+{ // Verify unadulterated source is loaded when there are no loaders
+  const { status, stdout } = spawnSync(
+    process.execPath,
+    [
+      ...commonArgs,
+    ],
+    { encoding: 'utf8' },
+  );
+
+  assert.strictEqual(status, 0);
+  assert.match(stdout[0], /\{/); // node:fs is an object
```

Comment:
This is not suuuper, but the alternatives I could think of (ex `assert.strictEqual(JSON.parse(stndout), fs)`) did not seem worth while.